src/components/modal/v2/styles/components/_offer-accordion.scss

```
@@ -22,6 +22,31 @@
             outline: 3px solid $light-blue;
             border-color: transparent;
         }
+        &.v5Design {
+            width: 100%;
+            padding-right: 5px;
+            background-color: $ql-gray;
+            .accordion__row {
+                button {
+                    border-radius: 11px;
+
+                }
+            }
+        }
+
+        &.open {
+            &.v5Design {
```

```suggestion
            &.v5Design {
            outline: none;
```
Something I missed in my original review - I noticed a thin white outline around the offer boxes for DE and this should fix it.